src/components/Search/SearchDropdown.tsx

```diff
@@ -21,7 +21,7 @@ function SearchDropdown(props: SearchDropdownProps) {
             id={repoMap[key]}
             value={repoMap[key]}
             key={repoMap[key]}
-            onClick={(e) => handleChangeDropdown(e)}
+            onClick={(e) => void handleChangeDropdown(e)}
```

Comment:
`Promise-returning function provided to attribute where a void return was expected  @typescript-eslint/no-misused-promises`

https://typescript-eslint.io/rules/no-misused-promises/